### PR TITLE
Optimize and refactor code from register view and invitational code

### PR DESCRIPTION
**Pull request by Andresws12**

### CODE REVIEW

**Comment by Andresws12** [comment]
@PVillasana @gonzalo-plaza @JuanJosePG Could do you review the code, please?

File: src/components/Login/MainRegisterForm.vue [hunk 1]
```diff
@@ -1,94 +1,57 @@
 <template>
-    <div>
-        <button-translation></button-translation>
+    <div class="main-reigster-form">
```
Comment (gonzalo-plaza) [inline src/components/Login/MainRegisterForm.vue:2]: @Andresws12 plase, fix the class name.

File: src/components/Login/MainRegisterForm.vue [hunk 2]
```diff
@@ -101,30 +64,39 @@ import { Component, Vue } from 'vue-property-decorator';
...
     private surname: string = '';
+
     private email: string = '';
+    private isEmailVaild: boolean = true;
+
     private password: string = '';
-    private isPasswordEmpty: boolean = false;
-    private isRepeatPasswordEmpty: boolean = false;
-    private passwordRepeat: string = '';
+    private isPasswordVaild: boolean = true;
```
Comment (gonzalo-plaza) [inline src/components/Login/MainRegisterForm.vue:93]: @Andresws12 fix the name of the variable please.

File: src/components/Login/MainRegisterForm.vue [hunk 3]
```diff
@@ -134,102 +106,55 @@ export default class MainRegisterForm extends Vue {
...
-        return this.password === this.passwordRepeat
-            ? null
-            : `${this.$t('components.register.notSamePassword')}`;
-    }
-    private get isNotRequiredLength(): string | null {
-        return this.password.length >= 8 || this.password.length === 0
-            ? null
-            : `${this.$t('components.register.notPasswordLength')}`;
+    private onCheckRepeatPassword(isPasswordVaild: boolean): void {
+        this.isRepeatPasswordVaild = isPasswordVaild;
```
Comment (gonzalo-plaza) [inline src/components/Login/MainRegisterForm.vue:133]: @Andresws12 Here is the same error with the word "Vaild"

File: src/components/Login/MainRegisterForm.vue [hunk 4]
```diff
@@ -134,102 +106,55 @@ export default class MainRegisterForm extends Vue {
...
-            this.passwordRepeat === '' ||
-            !this.isPasswordValid
-            ? true
-            : false;
+    private get isValidForm(): boolean {
+        return (
+            this.isEmailVaild &&
+            this.isPasswordVaild &&
+            this.isRepeatPasswordVaild
+        );
```
Comment (gonzalo-plaza) [inline src/components/Login/MainRegisterForm.vue:141]: @Andresws12 And here the same.

**Review by gonzalo-plaza — COMMENTED** [review]
@Andresws12 I don´t look more errors in your code.

**Comment by gonzalo-plaza** [comment]
@Andresws12 @PVillasana @pedrojgm @JuanJosePG For my this is okay to can be merged. @Andresws12 Please, when you fix the errors, mention me in a comment to can know that you have fix the error.

**Review by PVillasana — APPROVED** [review]

**Comment by Andresws12** [comment]
@PVillasana @gonzalo-plaza @JuanJosePG @pedrojgm I changed validations, please review the code, validations and layout

**Comment by gonzalo-plaza** [comment]
@Andresws12 Add padding in form of register in the touch view.

**Comment by gonzalo-plaza** [comment]
@Andresws12 the register form, need more padding in the touch view.

**Pull request merged**